src/main/java/teammates/ui/webapi/output/FeedbackSessionStatsData.java

```diff
@@ -0,0 +1,24 @@
+package teammates.ui.webapi.output;
+
+/**
+ * Api output of Feedback Session Status to hold information
```

Comment:
- Api -> `API`

- `Statistics` (or abbr.  stats) not `status`

- There is no need to have `to hold information of submitted feedback responses status`

Reply:
> There is no need to have `to hold information of submitted feedback responses status`

I thought Willson said don't just put "API output of { class }" in docs because this will not show what the format encapsulates

Reply:
I am not asking to change to `API output of { class }`. I saying you can delete `to hold information of submitted feedback responses status`.